wallet/src/account/txo_cache/mod.rs

```diff
@@ -0,0 +1,132 @@
+// Copyright (c) 2023 RBB S.r.l
+// opensource@mintlayer.org
+// SPDX-License-Identifier: MIT
+// Licensed under the MIT License;
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// https://github.com/mintlayer/mintlayer-core/blob/master/LICENSE
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use std::collections::{BTreeMap, BTreeSet};
+
+use common::{
+    chain::{OutPoint, OutPointSourceId, TxOutput},
+    primitives::Idable,
+};
+use wallet_types::{
+    account_id::AccountBlockHeight, wallet_block::WalletBlock, AccountTxId, WalletTx,
+};
+
+pub struct TxoCache {
```

Comment:
Please explain why this was created and what purpose it serves.

Also... why call it TxoCache instead of OutputCache?